lib/src/components/NxDrawer/NxDrawer.scss

```diff
@@ -0,0 +1,189 @@
+  }
+}
+
+.nx-drawer {
+  // Undo styles that chrome applies to <dialog> elements
+  border: initial;
+  height: initial;
+  left: initial;
+  margin: initial;
+  max-height: initial;
+  max-width: initial;
+  padding: initial;
+  width: initial;
+
+  // Prevent brief flash of scrollbar as tooltips disappear
+  overflow: hidden;
+
+  &::backdrop {
+    // Chrome tries to add its own subtle mask here which doubles up with our own
+    display: none;
+    pointer-events: auto;
+  }
+
+  grid-area: content;
+  box-sizing: border-box;
+  position: relative;
+  background-color: transparent;
+  color: var(--nx-swatch-grey-30);
+  justify-self: end;
+  width: 100%;
+  pointer-events: none;
+  z-index: var(--nx-z-index-drawer);
+}
+
+.nx-drawer__animation-wrapper {
+  display: flex;
+  height: 100%;
+  position: absolute;
+  right: 0;
+  top: 0;
+
+  // Prevents flickering
+  backface-visibility: hidden;
+  animation: slide-in 200ms ease-out;
+
+  &--close {
+    animation: slide-out 180ms ease-in;
+  }
+
+  @media (prefers-reduced-motion) {
+    &, &--close {
+      // We are setting the duration to 1ms
+      // because 0 did not trigger the animationEnd event.
+      animation-duration: 1ms;
+    }
+  }
+}
+
+.nx-drawer__panel {
+  box-sizing: border-box;
+  display: flex;
+  align-items: stretch;
+  flex-direction: column;
+  pointer-events: auto;
+  background-color: var(--nx-swatch-white);
+  position: relative;
+  padding: var(--nx-spacing-6x);
+  opacity: 1;
+  min-height: 0;
+  width: 520px;
+
+  &--narrow {
+    width: 348px;
+  }
+
+  &::after {
+    content: " ";
+
+    box-shadow: -2px 0px 50px rgba(128, 128, 128, 0.5);
+    height: calc(100% - var(--nx-spacing-16x));
+    width: 100%;
+    position: absolute;
+    left: 0;
+    top: var(--nx-spacing-8x);
+    z-index: -1;
+  }
+
+  .nx-form {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    height: 100%;
+    min-height: 0;
+  }
+
+  .nx-drawer-content {
+    --nx-drawer-padding: var(--nx-spacing-6x);
+
+    @include nx-container-helpers.container-vertical;
+
+    overflow: auto;
+    padding: var(--nx-spacing-6x);
+    height: 100%;
+    margin-right: calc(0px - var(--nx-drawer-padding));
+    margin-left: calc(0px - var(--nx-drawer-padding));
+    margin-bottom: calc(0px - var(--nx-drawer-padding));
+  }
+
+  .nx-footer {
+    @include nx-container-helpers.container-vertical;
+
+    border-top: 1px solid var(--nx-swatch-indigo-90);
+    height: min-content;
+    flex-shrink: 0;
+  }
+}
+
+.nx-drawer-header {
+  @include nx-container-helpers.container-vertical;
+
+  border-bottom: 1px solid var(--nx-swatch-indigo-90);
+  display: grid;
+  column-gap: var(--nx-spacing-2x);
+  grid-template: "title cancel-button"
+                 "subtitle subtitle"
+                 "description description" / auto min-content;
+  padding-bottom: var(--nx-spacing-6x);
+  flex-shrink: 0;
+
+  &__cancel-button, &__title, &__subtitle, &__description {
+    margin: 0;
```

Comment:
I'm not a big fan of specifying things only to immediately override them.  The subtitle and description have their own non-zero margins defined below, and so should not be included here

Reply:
> I'm not a big fan of specifying things only to immediately override them. The subtitle and description have their own non-zero margins defined below, and so should not be included here

Fixed here: 78d8ac4ca83c2142ac8a016b59bb89e0257f4968